gensrc/thrift/MasterService.thrift

```diff
@@ -46,6 +46,7 @@ struct TTabletInfo {
     14: optional bool is_in_memory
     15: optional bool enable_persistent_index
     16: optional Types.TVersion min_readable_version
+    17: optional i64 binlog_version
```

Comment:
What's the `binlog_version` ?

Reply:
Maybe binlog_config_version is better,  because open or close binlog is  best offort, BE needs to bring the the version of binlog config information when reporting tablet info, FE will synchronize the latest binlog config if version less than the it in FE